thriftbp/headers.go

```diff
@@ -44,3 +45,27 @@ func AddClientHeader(ctx context.Context, key, value string) context.Context {
 	headers = append(headers, key)
 	return thrift.SetWriteHeaderList(ctx, headers)
 }
+
+// Header gets the value of a thrift header by key
+//
+// If the value is not present we fall back to a case-insensitive check
+func Header(ctx context.Context, key string) (v string, ok bool) {
```

Comment:
I don't think we want to export this. if a service code need to do the same thing they can just duplicate this function.